scripts/dex-router/package.json

```diff
@@ -14,7 +14,7 @@
     "@coral-xyz/anchor": "^0.30.1",
```

Comment:
also let's start bumping the versions on this. we should start to stick to some standard, so if a PR is a backward incompatible feature addition, we should increment the minor version pre-release. here i think it's not, so can probs just patch bump